# Joint angles for one side (left or right)
# All angles in degrees

float32 shoulder_abduction
float32 shoulder_extension
float32 elbow_flexion
